### DISCUSSION TITLE
smart meter script not parsing correctly?

### DISCUSSION THREAD

**Discussion by evlo** [arendst/Tasmota#16207] [Q&A] [answered]
I have issue with smart meter script that 
value for 1.8.0 is overwritten by 61.8.0, as it also contains 1.8.0

what is wrong?

![image](https://user-images.githubusercontent.com/8813196/184213300-bd09bc6c-1ed8-4fd0-8655-092031988a18.png)

but it is workaroundable by t1+t2 or l1+l2+l3

**Comment by gemu2015** [comment, marked as answer]
try to compile with
#define SML_OBIS_LINE
should solve this problem
